site/content/docs/contributing/getting-started.md

```diff
@@ -118,8 +127,8 @@ contributor guides, signed the CLA ... now what?
 
 If you're planning to contribute code changes, you'll want to read the [development guide] next.
 
-If you plan to contribute documentation changes, you'll specifically want to see
-the [documentation section] of the development guide.
+If you're looking to contribute documentation improvements, first: Thank you! 🎉🤗
+You'll specifically want to see the [documentation section] of the development guide.
 
```

Comment:
just noticed a missing link for [CNCF] on line 30

Reply:
/hold

Reply:
fixed
/hold cancel

Reply:
/lgtm